gallery_dl/extractor/mangafox.py

```diff
@@ -60,3 +64,60 @@ def images(self, page):
 
             pnum += 2
             page = self.request("{}/{}.html".format(self.urlbase, pnum)).text
+
+
+class MangafoxMangaExtractor(MangaExtractor):
+    """Extractor for manga from fanfox.net"""
+    category = "mangafox"
+    root = "https://m.fanfox.net"
+    chapterclass = MangafoxChapterExtractor
+    pattern = BASE_PATTERN + r"(/manga/[^/]+)$"
+    test = (
+        ("https://fanfox.net/manga/kanojo_mo_kanojo", {
+            "pattern": MangafoxChapterExtractor.pattern,
+            "count": ">=60",
+        }),
+        ("https://mangafox.me/manga/shangri_la_frontier", {
+            "pattern": MangafoxChapterExtractor.pattern,
+            "count": ">=45",
+        }),
+        ("https://m.fanfox.net/manga/sentai_daishikkaku"),
+    )
+
+    def __init__(self, match):
+        MangaExtractor.__init__(self, match)
+
+    def chapters(self, page):
+        results = []
+        data = self.parse_page(page, {"lang": "en", "language": "English"})
+
+        pos = page.index('<dd class="chlist">')
+        while True:
+            url, pos = text.extract(page, '<a href="//', '"', pos)
+            if url == 'mangafox.la?f=mobile':
+                return results
+            info, pos = text.extract(page, '>', '<span', pos)
+            date, pos = text.extract(page, 'right">', '</span>', pos)
+
+            match = re.match(r"Ch (\d+)(\S*)(?: (.*))?", text.unescape(info))
+            if match:
+                chapter, minor, _ = match.groups()
```

Comment:
Why capture a string if you're gonna throw it away anyway ?

Reply:
The last part contains the chapter title. Since I haven't touched it yet (the chapter extractor doesn't support title metadata right now), I accidentally left it there.

Reply:
Ah ok. Makes sense.

But even in that case you don't need to nest a capturing group inside a non-capturing one when you only have a space before the title.